package.json

```
@@ -11,7 +11,8 @@
     "link": "npm link",
     "unlink": "npm unlink cherry",
     "start": "npm run build && ./bin/cherry",
-    "start:dev": "npm run build:dev && NODE_ENV=development ./bin/cherry"
+    "start:dev": "npm run build:dev && NODE_ENV=development ./bin/cherry",
+    "tests": "elm-test"
```

I think it's a bit more idiomatic for these to all be verbs. Obviously could just add both if you want to use `npm tests` as well
```suggestion
    "test": "elm-test"
```